crates/hyperqueue/src/server/client/autoalloc.rs

```diff
@@ -213,3 +233,41 @@ fn create_queue(
         Err(err) => ToClientMessage::Error(format!("Could not create autoalloc queue: {}", err)),
     }
 }
+
+async fn try_submit_allocation(
+    manager: ManagerType,
+    params: AllocationQueueParams,
+) -> anyhow::Result<()> {
+    let tmpdir = TempDir::new("hq")?;
+    let mut handler =
+        create_allocation_handler(&manager, params.name.clone(), tmpdir.as_ref().to_path_buf())?;
+    let worker_count = params.workers_per_alloc;
+    let queue_info = create_queue_info(params);
+
+    let allocation = handler
+        .submit_allocation(0, &queue_info, worker_count as u64)
+        .await
+        .map_err(|e| anyhow::anyhow!("Could not submit allocation: {:?}", e))?;
+
+    let working_dir = allocation.working_dir().to_path_buf();
+    let id = allocation
+        .into_id()
+        .map_err(|e| anyhow::anyhow!("Could not submit allocation: {:?}", e))?;
+    let allocation = Allocation {
+        id: id.to_string(),
+        worker_count: 1,
+        queued_at: SystemTime::now(),
+        status: AllocationStatus::Queued,
+        working_dir,
+    };
+    handler
+        .remove_allocation(&allocation)
+        .await
+        .map_err(|e| anyhow::anyhow!("Could not cancel allocation {}: {:?}", allocation.id, e))?;
+
+    log::info!(
+        "Allocation was submitted successfully. It was immediately canceled to avoid wasting
```

Comment:
Maybe but an adjective before "allocation" to clarify the message? (Something "A probing allocation was ..." ?)